generative_ai/function_calling/test_function_calling.py

```diff
@@ -52,12 +55,14 @@ def test_function_calling_advanced_function_selection() -> None:
     )
 
 
+@pytest.mark.skip(reason="Blocked on b/... ")
```

Comment:
Both samples that use OpenAI client are **not working** anymore ([log](https://btx.cloud.google.com/invocations/d7d20dd6-a816-43dc-8513-51f897b73d7c/log)):
```
 File "/workspace/generative_ai/function_calling/.nox/py-3-9/lib/python3.9/site-packages/openai/_client.py", line 122, in __init__
    super().__init__(
  File "/workspace/generative_ai/function_calling/.nox/py-3-9/lib/python3.9/site-packages/openai/_base_client.py", line 825, in __init__
    self._client = http_client or SyncHttpxClientWrapper(
  File "/workspace/generative_ai/function_calling/.nox/py-3-9/lib/python3.9/site-packages/openai/_base_client.py", line 723, in __init__
    super().__init__(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'proxies'
- generated xml file: /workspace/generative_ai/function_calling/sponge_log.xml -
=========================== short test summary info ============================
FAILED test_function_calling.py::test_function_calling_basic - TypeError: __i...
FAILED test_function_calling.py::test_function_calling_config - TypeError: __...
========================= 2 failed, 5 passed in 14.85s =========================
```

Reply:
Created a general [issue](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/12990) to initiate the discussion about current CI process.